lib/services/data/award/award_db_repository.dart

```diff
@@ -27,6 +33,13 @@ mixin AwardDbRepository implements DbRepository {
 		return dbClient.queryOneTyped(Collection.user, query, (json) => (Caregiver.fromJson(json)).badges);
 	}
 
+	Future createChildReward(ObjectId childId, ChildReward reward) {
+		var document = modify;
+		if (reward != null)
+			document.addToSet('rewards', reward.toJson());
```

Comment:
Don't think we need `addToSet` here (and it's not going to work if we would want to add two identical rewards at the same time) - simple `push` should do it. Same thing for `createBadge` probably.